### PR TITLE
Add documentation for auxiliary/scanner/redis/redis_server module

### PR DESCRIPTION
**Pull request by nsa**
This commit adds a documentation for `auxiliary/scanner/redis/redis_server` module.

Related to https://github.com/rapid7/metasploit-framework/issues/12389

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/redis/redis_server.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- Redis 4.0.14
+
+
+## Verification Steps
+
+  1. Do: `use auxiliary/scanner/redis/redis_server`
+  2. Do: `set rhosts [ips]`
+  3. Do: `set command [redis command]`
+  4. Do: `run`
+
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/redis/redis_server.md]: Please add the options header, and include the command option. Make sure to label the default!
If there is a website which lists valid commands, that would be good to link here

Comment (nsa) [reply documentation/modules/auxiliary/scanner/redis/redis_server.md]: I have no time now but I'll do in a few days. Thank you for your review.

**Comment by h00die** [comment]
Thanks for the docs!

**Comment by h00die** [comment]
# Release Notes
This PR adds docs for the redis_server aux scanner module

**Pull request merged**